File: SpringBootProject/Backend/ReadyRome/src/main/java/com/tour/entities/BookingDetails.java
package com.tour.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString(callSuper = true, exclude = {"user", "Mypackage","cart"})
public class BookingDetails extends BaseEntity {

	@NotNull
	private String name;
	@NotNull
	private String address;
	@NotNull
	private Integer age;
	@NotNull	
	private String addharNo;
	private String passportNo;
	@NotNull
	private String contactNo;
	
	@ManyToOne
	@JoinColumn(name="userId")
	private User user;
	
	@ManyToOne
	@JoinColumn(name="pkgId")
	private MyPackage myPackage;
	
}

File: DotNetProject/Tourism/Tourism/Entities/BookingDetails.cs
using System.Text.Json.Serialization;

namespace Tourism.Entities
{
    public class BookingDetails:BaseEntity
    {
        public int PkgId { get; set; }
        public int UserId { get; set; }
        public string Name { get; set;}
        public string Email { get; set; }
        public string Address { get; set; }
        public int Age {  get; set; }
        public string AddharNo { get; set; }
        public string PassportNo { get; set; }
        public string ContactNo { get; set; }
        [JsonIgnore]
        public User? user { get; set; }
        [JsonIgnore]
        public MyPackage? myPackage { get; set; }

    }
}
